components/organisms/o-bottom-navigation.vue

```diff
@@ -31,7 +32,7 @@ export default {
     return {
       navigationItems: [
         { icon: 'home', label: this.$t('Home'), onClick: this.goToHome },
-        { icon: 'search', label: this.$t('Search'), onClick: this.goToSearch },
```

Comment:
After removing `Search` item from bottom navigation, searching on mobile view will not be possible  🤔

Reply:
you are right, I made update and now search is also available in bottom nav